packages/client/auth-core/src/contexts/authlogin.tsx

```diff
@@ -17,14 +17,22 @@ const AuthContext = createContext<AuthContextType>({
     jwt: null,
 });
 
-type AuthProviderParams = {
+export type AuthProviderParams = {
     apiKey: string;
     environment: CrossmintEnvironment;
     children: ReactNode;
 };
 
+const getJwtFromCookie = (): string | null => {
+    if (typeof document === "undefined") {
+        return null; // Check if we're on the client-side
+    }
+    const crossmintSession = document.cookie.split("; ").find((row) => row.startsWith("crossmint-session"));
+    return crossmintSession ? crossmintSession.split("=")[1] : null;
+};
+
```

Comment:
This function helps set the jwt on initial render right away rather than having to make a call, then set it afterwards.